src/hera/cron_workflow.py

```diff
@@ -41,6 +41,8 @@ class CronWorkflow:
         The name of the service account to use in all workflow tasks.
     labels: Optional[Dict[str, str]] = None
         A Dict of labels to attach to the CronWorkflow object metadata
+    namespace: Optional[str]
```

Comment:
If we end up changing the type to a primitive one exclusively, then this can also work:
```suggestion
    namespace: str = 'default'
```

Reply:
Never mind, actually. I see the usages on `create/suspend/resume` 🙂 
```suggestion
    namespace: Optional[str] = None
```